CHANGELOG.md

```diff
@@ -3,102 +3,107 @@
 `co-log-core` uses [PVP Versioning][1].
 The change log is available [on GitHub][2].
 
+## 0.3.3.0 — Nov 2, 2022
```

Comment:
Need to adapt date before release is made.

Reply:
Good catch!